### ISSUE TITLE
Must use PackageReference

### ISSUE THREAD

**Issue by wojciech-pawlik** [Kinnara/ModernWpf#304]
I configured it exactly like in the description:
- created new WPF App (.net framework)
- installed package from NuGet: `Install-Package ModernWpfUI`
- edited App.xaml and MainWindow.xaml (like in Quick Start)

Unfortunately, I cannot build the project. I get:
![image](https://user-images.githubusercontent.com/32222222/110225186-221e0200-7ee3-11eb-999c-cf9ae9fb9096.png)

It does not refer to any file or specified fragment of the code.

**Comment by av1la** [comment]
right-click `packages.config` -> `Migrate packages.config to PackageReference`

**Comment by wojciech-pawlik** [comment]
Yes, I tried it - but I cannot configure it this way:
![image](https://user-images.githubusercontent.com/32222222/110248758-0104f200-7f73-11eb-9d03-c53ef2d20995.png)

There is a way to add a reference like in description:
`<PackageReference Include="ModernWpfUI" Version="0.9.3" />`
Where should I insert it? Will it help?

**Comment by oSethoum** [comment]
Try using the second one which does not require the package reference ( WinRt dependency removed ) 

![image](https://user-images.githubusercontent.com/46224889/110389277-97fda700-8064-11eb-9852-0738ed338667.png)

**Comment by wojciech-pawlik** [comment]
For now, it works. Thanks to both of you.

In the meantime, I got another issue (by installing packages):
`Install-Package : Unable to find fallback package folder ModernWpfUis`

I found out the solution (link below) but it was like a kind of magic for me.
https://stackoverflow.com/questions/48333699/unable-to-open-the-nuget-package-manager-console

After it, I dealt with the original issue and can use ModernWpfUI but I can't describe the main reason...

**Comment by vsnthdev** [comment]
If migrated to PackageReference, although the UI is rendered normally during runtime. The WPF designer seems to ignore the styles. We get the following warnings 👇

![Snip](https://user-images.githubusercontent.com/24322511/135400977-442d0d2b-d6e8-4940-8462-1cee6ad78249.png)

A fix to this is absolutely appreciated 🙂
